pkg/host/internal/sriov/sriov.go

```diff
@@ -115,12 +115,15 @@ func (s *sriov) GetVfInfo(pciAddr string, devices []*ghw.PCIDevice) sriovnetwork
 		VfID:       id,
 	}
 
-	if mtu := s.networkHelper.GetNetdevMTU(pciAddr); mtu > 0 {
-		vf.Mtu = mtu
-	}
 	if name := s.networkHelper.TryGetInterfaceName(pciAddr); name != "" {
-		vf.Name = name
-		vf.Mac = s.networkHelper.GetNetDevMac(name)
+		link, err := s.netlinkLib.LinkByName(name)
+		if err != nil {
+			log.Log.Error(err, "getVfInfo(): unable to get VF Link Object", "name", name, "device", pciAddr)
+		} else {
+			vf.Name = name
+			vf.Mtu = link.Attrs().MTU
+			vf.Mac = string(link.Attrs().HardwareAddr)
```

Comment:
This construction will not work. You should use `link.Attrs().HardwareAddr.String()`

Reply:
IDE autocompleted it :(. yes i will fix. which probably related to CI failure.